src/main/js/Edit/FileEdit.tsx

```diff
@@ -389,6 +390,10 @@ class FileEdit extends React.Component<Props, State> {
       return <Notification type="danger">{t("scm-editor-plugin.edit.notEditable")}</Notification>;
     }
 
+    const extensionsProps = {
+      path: this.isEditMode() ? this.props.path : this.state.path + "/" + this.state.file?.name
+    };
```

Comment:
We need more context here to be useful for other plugins:

* repository
* the whole file
* revision
* and the path